src/Application/Messenger/Handler/SendRequestConsumer.php

```diff
@@ -98,4 +106,15 @@ private function fail(string $id, string $reason)
         $this->logger->error("Sending ID $id failed: $reason");
         throw new RuntimeException($reason);
     }
+
+    /**
+     * @param object $email
+     * @param string $fileName
+     * @return Swift_Image returns the embeded image
```

Comment:
@tbg-jasonfung in modern PHPs we can + should specify a real return type. This reveals errors, e.g. in this case we are returning a string, which make docs more accurate.